app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float, int]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
```

Comment:
Just for future 
![image](https://github.com/mate-academy/py-distance-class/assets/102595649/80e2886f-844c-4a9d-afd5-5c4f3f3b9486)